rospbt/ros1/generators/builtin_msg_field_types.py

```diff
@@ -285,3 +290,25 @@ def float64(min_value=FLOAT64_MIN_VALUE, max_value=FLOAT64_MAX_VALUE, allow_nan=
         Strategy with preconfigured default values.
     """
     return floats(min_value, max_value, allow_nan, allow_infinity)
+
+
+def date(min_value=DATE_MIN_VALUE, max_value=DATE_MAX_VALUE):
```

Comment:
Either (A) we should put all data types (message and parameter data types) into one file lets say `rospbt/ros1/generators/data_types.py`. Or (B) we should keep them in separate files `rospbt/ros1/generators/message_types.py` (current `rospbt/ros1/generators/builtin_msg_fieldd_types.py`) and `rospbt/ros1/generators/parameter_types.py`. As (B) would imply much duplication I would go for (A). Until there is no `v1.0.0` release we can still rename the file `rospbt/ros1/generators/builtin_msg_fieldd_types.py` later. Then we should put some note into the docs in case it is a message or parameter only type like for `date`. Could you add a note into the docstring that `date` is a parameter only type?

Reply:
Right, this can still be decided before `v1.0.0` release.